src/NuGet.Clients/NuGet.VisualStudio.Implementation/SolutionExplorer/Models/AssetsFileTargetLibrary.cs

```diff
@@ -85,6 +90,18 @@ static bool IsPlaceholderFile(string path)
 
                 return false;
             }
+
+            static bool IsDocumentationFile(string path)
+            {
+                // Content files are not considered documentation. They are displayed via different means.
+                if (path.StartsWith("contentFiles/", StringComparison.OrdinalIgnoreCase))
+                    return false;
+
+                return path.StartsWith("doc/", StringComparison.OrdinalIgnoreCase)
```

Comment:
doc, documentation folders not really established patterns, and imo, we should not encourage them.  The idea is that if we encourage and unmonitored folder, we can't really monitor it without breaking customers. 

IMO, it's enough to look at .txt, .md.

Reply:
Makes sense. I can update that. I searched my disk for `project.assets.json` files containing `.txt` and `.md` and found a bunch of packages that had such folders.

Reply:
Fixed in 65325da213ed8668a27085f4e6a84e177e46e1a6.